src/fdb5/types/TypeClimateMonthly.cc

```diff
@@ -34,7 +34,11 @@ TypeClimateMonthly::~TypeClimateMonthly() {
 
 static int month(const std::string &value) {
     if (isdigit(value[0])) {
-        eckit::Date date(value);
+        int n = stoi(value);
```

Comment:
Can we check that the string length <= 2 here. Otherwise we will be doing a lot of extra string work in most cases.